tests/test_result_table.py

```diff
@@ -0,0 +1,231 @@
+import numpy as np
+import os
+import tempfile
+import unittest
+
+from astropy.table import Table
+from pathlib import Path
+
+from kbmod.result_table import ResultTable
+from kbmod.search import Trajectory
+from kbmod.trajectory_utils import make_trajectory
+
+
+class test_result_table(unittest.TestCase):
+    def setUp(self):
+        self.num_entries = 10
+        self.trj_list = [
+            make_trajectory(x=i, y=i + 0, vx=i - 2.0, vy=i + 5.0, flux=5.0 * i, lh=100.0 + i, obs_count=i)
+            for i in range(self.num_entries)
+        ]
+
+    def test_create(self):
+        table = ResultTable(self.trj_list)
+        self.assertEqual(len(table), self.num_entries)
+        for i in range(self.num_entries):
```

Comment:
I feel like maybe dumping something into `utils.py` in the tests that just mocks out a few of these results could be helpful to remove a lot of this repeated code (including comparison code). But I have to go teach so I hadnt paid that much attention to tests.